### ISSUE TITLE
[Bug] 如何在推理时输出隐藏层

### ISSUE THREAD

**Issue by owerbai** [InternLM/lmdeploy#2937]
### Checklist

- [ ] 1. I have searched related issues but cannot get the expected help.
- [ ] 2. The bug has not been fixed in the latest version.
- [ ] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

我期望获取output的hidden_state和logits，logits看上去是可行的，不知道hidden_state是否有获取方法？

### Reproduction

111

### Environment

```Shell
111111
```

### Error traceback

```Shell
1111
```

**Comment by alanMachineLeraning** [comment]
同求

**Comment by lvhan028** [comment]
等 v0.7.0 发布后，可以看看是否满足需求。
在 v0.7.0 中，可以输出 logits，以及最后一层的 hidden_states

**Comment by alanMachineLeraning** [comment]
预计什么时候发布呢

**Comment by alanMachineLeraning** [comment]
logits看上去是可行的，这个怎么输出？

**Comment by lvhan028** [comment]
本周会发布。到时可以在 pipeline.md 文档中查阅 logits, last_hidden_states 的获取方法
